src/extensions/walls/WallsApplicationCustomizer.ts

```diff
@@ -254,7 +247,7 @@ export default class WallsApplicationCustomizer extends BaseApplicationCustomize
       timeout -= intervalTime;
 
       if (timeout <= 0) {
-        if (scope.properties.logging === "true") {
+        if (this.properties.logging === "true") {
```

Comment:
Will using `this` work here? I think it will lose context of `this` and you might have to use `scope`.